fs2/src/main/scala/laserdisc/fs2/RedisConnection.scala

```diff
@@ -38,8 +38,8 @@ object RedisConnection {
 
   private[fs2] final object impl {
 
-    def send[F[_]: Applicative](sink: Sink[F, Byte])(implicit log: LogWriter[F]): Sink[F, RESP] =
-      _.evalMap(resp => log.debug(s"sending $resp") *> resp.pure)
+    def send[F[_]: Applicative: FlatMap](sink: Sink[F, Byte])(implicit log: LogWriter[F]): Sink[F, RESP] =
```

Comment:
`: Monad`? https://github.com/typelevel/cats/blob/master/core/src/main/scala/cats/Monad.scala

Reply:
Right, no reason not to. We use only `pure`